packages/components/select/src/select.tsx

```diff
@@ -95,7 +96,10 @@ function Select<T extends object>(props: Props<T>, ref: ForwardedRef<HTMLSelectE
             {!shouldLabelBeOutside ? labelContent : null}
             <div {...getInnerWrapperProps()}>
               {startContent}
-              <span {...getValueProps()}>{renderSelectedItem}</span>
+              <span {...getValueProps()}>
+                {renderSelectedItem}
+                <VisuallyHidden>,</VisuallyHidden>
```

Comment:
@jguddas the selected values are already joined by a "," https://github.com/nextui-org/nextui/pull/1431/files#diff-bdef9d0be5667754a2de1d88a28f30d57743ca21a6fb6f25cf8d1b2681987e98R86

Reply:
This is a visually hidden comma, it changes so the sr will get `Dog, Cat, Favorite Animals`  
instead of `Dog, Cat Favorite Animals`